quantum/split_common/transport.c

```diff
@@ -218,11 +218,11 @@ void transport_slave(matrix_row_t master_matrix[], matrix_row_t slave_matrix[])
 #    endif
 
 #    if defined(LED_MATRIX_ENABLE) && defined(LED_MATRIX_SPLIT)
-    memcpy((void *)i2c_buffer->led_matrix, (void *)led_matrix_eeconfig, sizeof(i2c_buffer->led_matrix));
+    memcpy((void *)i2c_buffer->led_matrix.raw, (void *)led_matrix_eeconfig.raw, sizeof(i2c_buffer->led_matrix.raw));
```

Comment:
Shouldn't it be this?
```suggestion
    memcpy((void *)&i2c_buffer->led_matrix.raw, (void *)&led_matrix_eeconfig.raw, sizeof(i2c_buffer->led_matrix.raw));
```

Reply:
I think, maybe, need to swap memcpy's src and dist.

Reply:
yeah, you're right.